docs/layout/dynamic-font-scaling.md

```diff
@@ -0,0 +1,205 @@
+:::
+
+Dynamic Font Scaling is already enabled by default on Android. Developers can enable it on iOS by setting the following CSS globally:
+
+```css
+html {
+  --ion-dynamic-font: var(--ion-default-dynamic-font);
+}
+```
+
+Under the hood, Ionic sets the following CSS on iOS devices to enable Dynamic Font Scaling:
+
+```css
+html {
+  font: var(--ion-dynamic-font);
+}
+```
+
+Developers should ensure that the [typography.css](./global-stylesheets#typographycss) file is imported.
+
+## Using Dynamic Font Scaling
+
+### Integrating Custom Components
+
+Developers can configure their custom components to take advantage of Dynamic Font Scaling by converting any pixel font sizes to use [rem units](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units#lengths). An easy way to convert from `px` to `rem` is to divide the pixel font size by the default browser font size, which is typically 16px.
+
+For example, if a component has a font size of `14px`, then this could be converted to `rem` by doing `14px / 16px = 0.875rem`.
+
+Developers should also audit their applications and make any other changes necessary to support larger font sizes. One thing to keep in mind is that the dimensions of your components may need to change to accommodate the larger font sizes. As a result, it may be necessary to change properties such as `width` and `height` to `min-width` and `min-height`. We also recommend having long text wrap to the next line instead of truncating to keep large text readable.
+
+Also note that any Ionic components that have had their font sizes overridden should also be updated to use `rem` units.
+
+### Custom Font Family
+
+We recommend using the default fonts in Ionic as they are designed to look good at any size and ensure consistency with other mobile apps. However, developers can use a custom font family with Dynamic Font Scaling via CSS:
+
+```css
+html {
+  --ion-dynamic-font: var(--ion-default-dynamic-font);
+  --ion-font-family: 'Comic Sans';
+}
+```
+
+### `em` units versus `rem` units
+
+Developers have two options for relative font sizes: `em` and `rem`.
+
+`em` units set the font size of an element relative to the font size of its parent.
+
+In the following example, the computed font size of `.child` is `40px` because it is a child of `.parent` (2em \* 20px = 40px).
+
+```css
+.parent {
+  font-size: 20px;
+}
+
+.child {
+  font-size: 2em;
+}
+```
+
+However, the `em` unit has a compounding effect which can cause issues. In the following example, the second `.child` element has a computed font size of `80px` since the font sizes compound.
+
+```html
+<div class="parent">
+  Parent element with 20px
+  <div class="child">
+    Child element with 40px
+    <div class="child">Child element with 80px</div>
+  </div>
+</div>
+```
+
+<div style={{ fontSize: '20px' }}>
```

Comment:
Should we put this in a playground demo with just a javascript example? I know you don't need ionic for this, so we could override the index file and keep it simple.

Reply:
I think that may be overkill since that's going to show you Angular, React, and Vue code snippets which really aren't need. The purpose of this is to demonstrate how part of CSS works.